### ISSUE TITLE
Proposals repository

### ISSUE THREAD

**Issue by chicoxyzzy** [WebAssembly/design#1235]
Hi. I just creates [WebAssembly proposals tracking repository](https://github.com/chicoxyzzy/WebAssembly-proposals) as I proposed on [September 18 CG meeting](https://github.com/WebAssembly/meetings/blob/master/2018/CG-09-18.md#review-of-action-items-from-prior-meeting).

Please let me know what you think about it. I'll try to keep it up-to-date. I'll be happy to transfer this repo to WebAssembly organization if it makes sense.

**Comment by jfbastien** [comment]
We already have this information here: https://webassembly.org/docs/future-features/
As well as through tracking issues: https://github.com/WebAssembly/design/issues?q=is%3Aissue+is%3Aopen+label%3A%22🛤+tracking%22
It's documented in step 7 of the process: https://github.com/WebAssembly/meetings/blob/master/process/phases.md
The presentation is nice, but I don't want us duplicating information more (and inevitably getting it out-of-date). Maybe send PRs to update what's there, and make it nicer?

**Comment by binji** [comment]
@jfbastien: I don't want to see duplication either, but we've had folks mention before that it's hard to follow how proposals are moving forward currently. Moving them into a repository would make this easier.

**Comment by jfbastien** [comment]
@binji works for me, all I'm saying is we shouldn't duplicate things even more. Ideally it would all auto-generate from the issue tracker 😉

**Comment by binji** [comment]
I added an agenda item to discuss this [here](https://github.com/WebAssembly/meetings/pull/319)

**Comment by chicoxyzzy** [comment]
@binji should we move 
https://github.com/chicoxyzzy/WebAssembly-proposals to WebAssembly organization or create new repository?

**Comment by binji** [comment]
I don't know how the permissions work for moving the project. Since there isn't much in https://github.com/chicoxyzzy/WebAssembly-proposals yet, let's create a new repo and we can move the content there.

OK done :-) : https://github.com/WebAssembly/proposals

**Comment by binji** [comment]
OK, I've moved the tracking issues to that repo, updated FutureFeatures.md to point to these new issues, and copied the content from https://github.com/chicoxyzzy/WebAssembly-proposals.

Anything I'm forgetting?

**Comment by chicoxyzzy** [comment]
Thank you. I think that's all for now.

**Comment by chicoxyzzy** [comment]
@binji should we add proposals repository as pinned repository of [WebAssembly organization](https://github.com/WebAssembly)?

**Comment by binji** [comment]
> @binji should we add proposals repository as pinned repository of WebAssembly organization?

Done.
